nautobot/docs/user-guide/administration/guides/prometheus-metrics.md

```diff
@@ -25,6 +25,9 @@ CACHES = {
 }
 ```
 
++++ 2.2.1
+    In case the `/metrics` endpoint is unacceptably slow to respond, you can disable specific apps that may be causing problems with the `METRICS_DISABLED_APPS` configuration setting.
```

Comment:
```suggestion
    In case the `/metrics` endpoint is not performant or not required slow, you can disable specific apps with the `METRICS_DISABLED_APPS` configuration setting.
```